assets/training/model_management/src/run_model_download.py

```diff
@@ -99,18 +88,17 @@ def run():
     logger.info(f"Update existing model: {update_existing_model}")
 
     if update_existing_model == "false":
-        validate_if_model_exists(model_id)
+        if validate_if_model_exists(model_id):
+            return False
```

Comment:
Should we have a false scenario to download model from registry and registration details and setting that to pipeline output?

Reply:
we would need a new component to do that.